app/assets/stylesheets/sbstyles.scss

```diff
@@ -19,8 +22,51 @@ h2 {
 
 .top-bar {
   margin-bottom: 10px;
+  box-shadow: 0 1px $tertiary-green;
+  background: linear-gradient($primary-green, $secondary-green);
 }
 
 .button.tiny.radius {
   background-color: $review-button-color;
 }
+
+.bg {
+  min-height: 100%;
```

Comment:
Properties should be ordered height, left, min-height, min-width, position, top, width